docker-compose.yml

```diff
@@ -19,21 +19,24 @@
 
 web:
   build: .
-  command: inveniomanage runserver -h 0.0.0.0 -p 28080
+  command: /code/scripts/docker_boot.sh inveniomanage runserver -h 0.0.0.0 -p 28080
   ports:
-    - "28080:28080"
+    - "127.0.0.1:28080:28080"
   environment:
     - CACHE_REDIS_HOST=cache
     - CFG_DATABASE_HOST=db
     - INVENIO_APP_CONFIG_ENVS=CACHE_REDIS_HOST,CFG_DATABASE_HOST
   volumes:
```

Comment:
See above, plus: Do not mirror the code for the production setup. So lines like `foo:bar[:ro]` should never appear in 'docker-compose.yml'